src/View.php

```diff
@@ -455,7 +455,7 @@ public function beforeRender(string $viewFile, array $parameters): bool
      * @param string $output the rendering result of the view file. Updates to this parameter
      * will be passed back and returned by {@see renderFile()}.
```

Comment:
```suggestion
     * @param string $output the rendering result of the view file.
    * @return string Updated output. It will be passed to {@see renderFile()} and returned.
```